sdk/monitor/azure-monitor-opentelemetry-exporter/tests/quickpulse/test_utils.py

```diff
@@ -425,3 +456,186 @@ def test_get_metrics_from_projections(self, projection_map_mock):
                 ("test-id4", 4.0),
             ],
         )
+
+    def test_validate_info_invalid_telemetry_type(self):
+        valid_metric_info = mock.Mock(telemetry_type="Request", projection=None, filter_groups=[])
```

Comment:
naming a little confusing here - this is testing for invalid scenarios, but the derivedMetricInfo is named valid_